src/gitops/components/application/ApplicationResourcesView.tsx

```
@@ -0,0 +1,402 @@
+import { t } from '@gitops/utils/hooks/useGitOpsTranslation';
+import { ApplicationKind, ApplicationResourceStatus } from '@gitops-models/ApplicationModel';
+import {
+  Action,
+  K8sGroupVersionKind,
+  ListPageFilter,
+  ResourceLink,
+  RowFilter,
+  RowFilterItem,
+  useListPageFilter,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { EmptyState, EmptyStateBody, Flex, FlexItem, Stack, StackItem } from '@patternfly/react-core';
```

ESLint

[error] Replace `·EmptyState,·EmptyStateBody,·Flex,·FlexItem,·Stack,·StackItem·` with `⏎··EmptyState,⏎··EmptyStateBody,⏎··Flex,⏎··FlexItem,⏎··Stack,⏎··StackItem,⏎`

(prettier/prettier)